libs/sdk-ui-kit/src/Dialog/HubspotConversionTouchPointDialog.tsx

```diff
@@ -14,11 +14,17 @@ import {
 export const HubspotConversionTouchPointDialog: React.FC<IHubspotConversionTouchPointDialogBaseProps> = (
     props,
 ) => {
+    const onDialogSubmit = () => {
+        const submitBtn = document.querySelector(".hs-button.primary.large") as HTMLInputElement;
```

Comment:
Can the default class name for submit button `.hs-button.primary.large` be changed in the future if we need to upgrade package `@aaronhayes/react-use-hubspot-form`?

Should we define a value for property `submitButtonClass` where we can set a custom css class for that button, then we can be sure that the class name won't change even when the package is upgraded?
https://github.com/aaronhayes/react-use-hubspot-form/blob/cb96d3d99c3b7bb24450eca995b6df5fe1416b7a/src/index.ts#L50

※ We might have to add a custom css name here https://github.com/gooddata/gooddata-ui-sdk/blob/559bbd5d7a571fff5899807b6fe8581022e3cc46/libs/sdk-ui-kit/src/Dialog/HubspotConversionTouchPointDialogBase.tsx#L157